frontend/src/Home.css

```diff
@@ -1,28 +1,12 @@
 .home {
-    height: 100vh - 64px;
-    padding: 32px;
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    align-items: center;
-}
-
-.home-new-game {
-    padding: 32px;
-    font-size: 24px;
-    background-color: #fff;
-    color: #b20738;
-    border: 1px solid #b20738;
-    text-transform: uppercase;
-    font-weight: bold;
-    text-decoration: none;
-    text-align: center;
-}
-
-.home > button:hover {
-    border-bottom-width: 2px;
+  height: 100vh - 64px;
```

Comment:
This should be `calc(100vh - 64px)` to support Firefox/Chrome.
On the other hand, I'm not sure if this should kept if we plan to support multiple rows of crosswords because this will cause crosswords to be cut off.